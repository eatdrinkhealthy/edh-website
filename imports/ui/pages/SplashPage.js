import React from "react";
import { styles } from "../styles";
import CompanyName from "../components/CompanyName";
import Pitch from "../components/Pitch";
import ComingSoon from "../components/ComingSoon";
import Logo from "../components/Logo";

const SplashPage = () => (
  <div className={styles.defaultPage.className}>
    <CompanyName />
    <Pitch />
    <ComingSoon />
    <Logo />
  </div>
);

export default SplashPage;
