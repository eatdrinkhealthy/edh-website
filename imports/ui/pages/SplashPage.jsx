import React from "react";
import Header from "../components/Header";
import Footer from "../components/Footer";
import Preview from "../components/Preview";

const SplashPage = () => (
  <div>
    <Header />
    <Preview />
    <Footer />
  </div>
);

export default SplashPage;
