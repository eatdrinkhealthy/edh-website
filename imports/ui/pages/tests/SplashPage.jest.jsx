/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
import React from "react";
import { createStore } from "redux";
import { Provider } from "react-redux";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import SplashPage from "../SplashPage";
import appReducer from "../../../state/reducers";

describe("<SplashPage />", function () {
  const testStore = createStore(appReducer);

  it("matches render snapshot", function () {
    const tree = renderer
      .create(
        <Provider store={testStore}>
          <MemoryRouter>
            <SplashPage />
          </MemoryRouter>
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("contains sections header, main, footer", function () {
    const wrapper = shallow(<SplashPage />);
    expect(wrapper.find("Header").length).toEqual(1);
    expect(wrapper.find("Main").length).toEqual(1);
    expect(wrapper.find("Footer").length).toEqual(1);
  });
});
