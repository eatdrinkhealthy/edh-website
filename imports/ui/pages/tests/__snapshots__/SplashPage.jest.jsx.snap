// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SplashPage /> matches render snapshot 1`] = `
<div>
  <div>
    <div
      className="row"
    >
      <div
        className="column"
      >
        <div
          className="app-header__top"
        >
          <a
            href="/"
          >
            <img
              alt="Company Logo"
              src="images/logo.png"
            />
          </a>
          <nav
            className="app-header-nav"
          >
            <ul
              className="menu align-center"
            >
              <li>
                <a
                  aria-current="true"
                  className="active"
                  href="/"
                  onClick={[Function]}
                  style={
                    Object {
                      "color": "#f28020",
                    }
                  }
                >
                  HOME
                </a>
              </li>
              <li>
                <a
                  aria-current={false}
                  className={undefined}
                  href="/privacy-policy"
                  onClick={[Function]}
                  style={undefined}
                >
                  PRIVACY
                </a>
              </li>
            </ul>
          </nav>
          <div
            className="app-header-social text-center medium-text-right"
          >
            <a
              href="mailto:info@eatdrinkhealthy.co"
            >
              <span
                className="fa-stack fa-lg"
              >
                <i
                  className="fa fa-circle fa-stack-2x"
                />
                <i
                  className="fa fa-envelope fa-stack-1x fa-inverse"
                />
              </span>
            </a>
            <a
              href="https://www.linkedin.com/company/16173480/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <span
                className="fa-stack fa-lg"
              >
                <i
                  className="fa fa-circle fa-stack-2x"
                />
                <i
                  className="fa fa-linkedin fa-stack-1x fa-inverse"
                />
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
    <header
      className="app-header"
    >
      <section
        className="app-header__banner"
      >
        <div
          className="app-header__banner-content-wrapper"
        >
          <div
            className="row"
          >
            <div
              className="column medium-8 large-7"
            >
              <div
                className="app-header__banner-headline"
              >
                A platform to find, share, and discuss healthy places to shop and eat.
              </div>
              <div
                className="app-header__banner-buttons"
              >
                <a
                  className="button button--app-store"
                  href="https://itunes.apple.com/app/id1116167404"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <i
                    className="fa fa-apple"
                  />
                  <span>
                    Download on
                    <br />
                    the 
                    <strong>
                      App Store
                    </strong>
                  </span>
                </a>
                <a
                  className="button button--wep-app"
                  href="https://app.eatdrinkhealthy.co/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <span>
                    Web App
                  </span>
                </a>
              </div>
            </div>
          </div>
        </div>
      </section>
    </header>
  </div>
  <main
    className="app-main"
  >
    <section
      className="app-main__content"
    >
      <div
        className="row"
      >
        <div
          className="column small-12 medium-7"
        >
          <img
            alt="Mobile Phone Device"
            src="images/device.png"
          />
        </div>
        <div
          className="column small-12 medium-5"
        >
          <div>
            <h2>
              Contact Us
            </h2>
            <form
              onSubmit={[Function]}
            >
              <div>
                <label
                  htmlFor="contactName"
                >
                  Name
                </label>
                <input
                  autoFocus={true}
                  component="input"
                  name="contactName"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  type="text"
                  value=""
                />
              </div>
              <div>
                <label
                  htmlFor="contactEmail"
                >
                  Email
                </label>
                <input
                  autoFocus={false}
                  component="input"
                  name="contactEmail"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  type="email"
                  value=""
                />
              </div>
              <div>
                <label
                  htmlFor="contactMessage"
                >
                  Message
                </label>
                <textarea
                  cols="10"
                  name="contactMessage"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  rows="50"
                  value=""
                />
              </div>
              <button
                className="button button--submit"
                id="contactSubmit"
                type="submit"
              >
                Send
              </button>
            </form>
          </div>
        </div>
      </div>
    </section>
  </main>
  <footer
    className="app-footer"
  >
    <div
      className="row"
    >
      <div
        className="column"
      >
        <img
          alt="Company Logo"
          src="images/carrot_icon.png"
        />
        <span
          className="app-footer__copyright"
        >
          <span>
            Copyright 2018 Eat. Drink. Healthy.
          </span>
          <span>
            All rights reserved.
          </span>
        </span>
      </div>
    </div>
  </footer>
</div>
`;
